### PR TITLE
Update events.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Fix link and rephrase for more precise content

### CODE REVIEW

File: docs/APIs/tasks/events.md [hunk 1]
```diff
@@ -144,4 +144,4 @@ We used `InterruptIn` for the example above, but the same kind of code can be us
...
+We have discussed only a small part of how event queues work in mbed OS. The `EventQueue` and `Event` classes in the `mbed-events` library offer a lot of features that this document does not cover, including calling functions with arguments, queueing functions to be called after a delay or queueing functions to be called periodically. The [README of the mbed-events library](https://github.com/ARMmbed/mbed-os/blob/master/events/README.md) shows more ways to use events and event queues. To see the implementation of the events library, review [the equeue library](https://github.com/ARMmbed/mbed-os/tree/master/events/equeue).
```
Comment (iriark01) [inline docs/APIs/tasks/events.md]: Can you check if this link is good
https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/equeue_8h_source.html

Comment (AnotherButler) [inline docs/APIs/tasks/events.md]: @geky Is the link from iriark01 OK to use?

Comment (geky) [inline docs/APIs/tasks/events.md]: If it's in the mbed-os tree and armmbed org it should be good 👍

Comment (geky) [inline docs/APIs/tasks/events.md]: Oh hey, if you change the link to focus on the first title it shifts the focus down to the readme:
https://github.com/ARMmbed/mbed-os/tree/master/events/equeue#the-equeue-library

As an added plus, if the readme is updated to break that tag, it'll still go to the page as normal

Comment (AnotherButler) [inline docs/APIs/tasks/events.md]: @geky What about this link? Can we use it, instead?
https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/equeue_8h_source.html

Comment (iriark01) [inline docs/APIs/tasks/events.md]: @geky <nudge>

Comment (geky) [inline docs/APIs/tasks/events.md]: Ah sorry. That link will work ok, at this point it's basically just pointing to the source code.

**Comment by iriark01** [comment]
I think you can copy this to 5.4 and publish for both 5.3 and 5.4

**Pull request merged**